examples/ai-core/src/stream-text/openai-web-search-tool.ts

```diff
@@ -23,7 +23,11 @@ async function main() {
       }
 
       case 'source': {
-        process.stdout.write(`\n\n Source: ${chunk.title} (${chunk.url})`);
+        if (chunk.sourceType === 'url') {
+          process.stdout.write(`\n\n Source: ${chunk.title} (${chunk.url})`);
+        } else {
+          process.stdout.write(`\n\n Document: ${chunk.title}`);
+        }
         break;
```

Comment:
can you add an example that tests the anthropic functionality end to end? we have a pdf in the examples that you can use